Fix some compatibility issues with GTA v1.67+

vCybah opened the pull request:
All I've tried to do with my fork is to make the script compatible with the new GTA version 1.67, as the interaction menu changed slightly, making some of the macros no longer work.

I'm not very good with AHK but I think this should be a working version for GTA v1.67+, everything I use regularly worked with my testing however there are a few things I couldn't test such as:

    have not tested manual inv settings
    have not tested CPH mode
    not sure what needs to be done with VIP toggle/check now that the interaction menu changed (so I just turned it off)

Things that needed changed slightly and should now work on v1.67:

    openOutfitMenu
    ToggleRadar
    ArmorMenu
    SnackMenu
    EquipScarf
    RetrieveCar
    RequestSparrow
    ReturnSparrow

Hope this helps a little, I'm sure you could have done all this in 2 seconds (and in a better way haha) but figured I'd do it just for now so anyone else who wants to download the script until it's fixed can still use it :)
I'm new to using GitHub still so Idrk how the Pull Req. stuff works If I need to do anything different just let me know, thanks.

2called-chaos commented:
Hello and thanks for the contribution! Sorry for late reply, I was on holidays :)

I'm currently not playing the game anymore (and don't have it installed) so it's hard for me to test things. Could you elaborate what changed with CEO mode? It used to shift the interaction menu by one (as CEO/MC options appeared), is that no longer the case?

I can merge this but you would need to remove the Readme from the PR (since you basically purged its contents). If you need help with that let me know and I'm happy to assist.

Again, thanks for the contribution

2called-chaos commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
 ps*.exe
 config.ahk
 custom.ahk
+config.ahk
```
The `config.ahk` is already ignored in line 2, doesn't hurt but it shouldn't be necessary, did your git client not ignore that file (just out of curiosity)?

vCybah replied:
Yeah, prob a noob error on that lol, at first I removed the config.ahk line from there while I was doing testing trying to see what needed to be changed and such and when I was done I added it back but I reverted it or something so I didn't realize it was there and it wanted me to add another so I did, I don't know exactly but once I noticed it I wasn't sure how to fix it so I just left it, plus I assumed that if you did want to implement this stuff you would have a better way of doing it so wasn't expecting it to be merged exactly but just wanting to let you know what changed and how I fixed it basically for if you or anyone else wanted to know so it can keep being used in the meantime :)

vCybah commented:
Sorry for the late response, I didn't get a notification for some reason and assumed that it was because there wasn't anything and I just got around to checking now. Also sorry for any mistakes/dumb things I do/say because I'm a noob and also sleep-deprived and about to go to sleep after I'm done here lol.

Now when you become a CEO instead of it moving in the menu like it did before the menu doesn't move at all, IIRC. It really wasn't much of anything that needed to be changed (at least as far as the stuff that I use in the script.) Mostly I was just trying to make that repo to get it to work for me with the new update while I was messing around on it and wanted to try to help out if I could but I'm not great with ahk or github lol.

Yeah, I can remove the readme if you want to do that like I said I'm new to GitHub and this stuff so I don't know exactly how pull requests work and such. I changed the readme so that it shows what I changed and stuff like that and I kinda assumed that if you merged it into yours you could just ignore that readme to copy over or something and anyone who saw mine would just click the link to get to the original readme on your repo, but ill just go ahead and delete it, no problem :)

Thanks again for this its great!

2called-chaos commented:
> I didn't get a notification for some reason

Yeah I have the same issue and the new Github UI is not really helping either :S

I restored the readme and merged it :) Thanks again
